concent_api/delete-unsupported-messages.py

```diff
@@ -0,0 +1,40 @@
+import django
+import os
+import sys
+
+from logging import getLogger
+
+from common.helpers import deserialize_message
+
+from golem_messages.exceptions import MessageError
+
+sys.path.append('concent_api')
+os.environ['DJANGO_SETTINGS_MODULE'] = "concent_api.settings"
+django.setup()
+
+from core.models import StoredMessage  # noqa E402
+from core.models import Subtask  # noqa E402
+
+logger = getLogger(__name__)
+
+
+def main() -> None:
+    for subtask in Subtask.objects.all():
+        try:
+            report_computed_task = deserialize_message(subtask.report_computed_task.data.tobytes())
+            if not isinstance(report_computed_task.size, int):
+                delete_unsupported_messages(subtask)
+        except MessageError as golem_messages_exception:
+            logger.info(f'During message deserialization exception raised: {golem_messages_exception}')
+            delete_unsupported_messages(subtask)
+
+
+def delete_unsupported_messages(subtask: Subtask) -> None:
+    subtask_id = subtask.subtask_id
+    StoredMessage.objects.filter(subtask_id=subtask_id).delete()
+    subtask.delete()
+    logger.info(f'Deleted subtask and all related messages with subtask_id: {subtask_id}')
```

Comment:
Now when I think about it.. do we really want to remove all related `StoredMessages` only because we were not able to deserialize `report_computed_task`? 

@cameel What do you think?

Reply:
Yes. If we're removing the subtask we don't care about the messages either.

We we cared about them we should have kept running the old version until the client fetched them.

Reply:
> We we cared about them we should have kept running the old version until the client fetched them.

But those are not messages which are returned to client (those are `PendindResponse` objects), but those which we received from client.